olm/README.md

```diff
@@ -0,0 +1,27 @@
+# Creating new OLM artifacts
+
+Script `update-nightly-olm-files.sh` is used for that
+
+Pre-requisites
+ - [yq/https://pypi.org/project/yq](https://pypi.org/project/yq) and not [http://mikefarah.github.io/yq/](http://mikefarah.github.io/yq/)
+ - [Operator SDK](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md)
```

Comment:
Could you mention that we should install `v0.8.2` instead of `v0.9.0` ?

Reply:
yes but usually people use the latest version of all products